### PR TITLE
Kotlin 1.3.40

### PR DESCRIPTION
**Pull request by CherryPerry**
**WIP**

Native compilation issue: https://github.com/JetBrains/kotlin-native/issues/3087
JS configuration issue: ~~https://youtrack.jetbrains.com/issue/KT-32164~~, ~~https://youtrack.jetbrains.com/issue/KT-31917~~ fixed in 1.3.50

See https://github.com/badoo/Reaktive/issues/123 for plan

### CODE REVIEW

File: buildSrc/build.gradle.kts [hunk 1]
```diff
@@ -9,7 +9,7 @@ repositories {
 
 dependencies {
     implementation("com.moowork.gradle:gradle-node-plugin:1.3.1")
-    implementation("org.jetbrains.kotlin:kotlin-gradle-plugin:1.3.31")
+    implementation("org.jetbrains.kotlin:kotlin-gradle-plugin:1.3.40")
```
Comment (arkivanov) [inline buildSrc/build.gradle.kts]: Can we reuse kotlin_version somehow?

Comment (CherryPerry) [reply buildSrc/build.gradle.kts]: Done in https://github.com/badoo/Reaktive/pull/126

**Comment by Archinamon** [comment]
any ETA here? would be great to have 1.3.40 support — it blocks all other native libraries updating to kotlin 1.3.40

**Comment by CherryPerry** [comment]
@Archinamon We are blocked by Kotlin 1.3.50 release.

I think you can try to fork and remove all JS support and add `-Xdisable-phases=Devirtualization` flag to native compilation. Probably should work.

**Comment by arkivanov** [comment]
@Archinamon are you actually using Reaktive in production? Would be nice to here some feedback so far.

**Comment by GianfrancoMS** [comment]
Kotlin 1.3.50 is out.

**Comment by arkivanov** [comment]
Yep, we'll update soon. @cherryperry FYI

**Comment by CherryPerry** [comment]
New PR: https://github.com/badoo/Reaktive/pull/171

**Pull request closed without merging**